tests/beaker_tests/cisco_interface/routedintflib.rb

```diff
@@ -66,6 +67,17 @@ def self.create_routedintf_manifest_switchport_disabled
       switchport_autostate_exclude => 'default',
       switchport_vtp               => 'default',
       vrf                          => 'default',
+  }}
+EOF"
+    manifest_str
+  end
+
+  def self.create_routedintf_manifest_channel_group
```

Comment:
channel-group prevents some L3 properties from being configured so I moved this into it's own test manifest.